src/test/integration/mocks/server.ts

```diff
@@ -0,0 +1,62 @@
+import assert from "assert";
```

Comment:
We are already using jest in this project, So maybe we shouldn't not introduce another mocking library?

Reply:
`assert` is for assertion.
the mocking lib that i am using here is `Sinon`.
we tried to run the test with jest but it didn't work, see discussion in vscode channel.
About the assertion lib we will probably use `jest expect` instead of `assert`, see discussion in vscode channel